src/libraries/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -471,7 +495,9 @@ public sealed partial class IncrementalHash : System.IDisposable
         public void AppendData(byte[] data, int offset, int count) { }
         public void AppendData(System.ReadOnlySpan<byte> data) { }
         public static System.Security.Cryptography.IncrementalHash CreateHash(System.Security.Cryptography.HashAlgorithmName hashAlgorithm) { throw null; }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
```

Comment:
Are we not marking this class as unsupported? I have these on the list I'm using

```
M:System.Security.Cryptography.IncrementalHash.GetHashAndReset(System.Span{System.Byte}),System.Security.Cryptography,IncrementalHash,GetHashAndReset(Span<Byte>),0
"M:System.Security.Cryptography.IncrementalHash.CreateHMAC(System.Security.Cryptography.HashAlgorithmName,System.ReadOnlySpan{System.Byte})",System.Security.Cryptography,IncrementalHash,"CreateHMAC(HashAlgorithmName, ReadOnlySpan<Byte>)",0
M:System.Security.Cryptography.IncrementalHash.GetCurrentHash(System.Span{System.Byte}),System.Security.Cryptography,IncrementalHash,GetCurrentHash(Span<Byte>),0
M:System.Security.Cryptography.IncrementalHash.AppendData(System.ReadOnlySpan{System.Byte}),System.Security.Cryptography,IncrementalHash,AppendData(ReadOnlySpan<Byte>),0
M:System.Security.Cryptography.IncrementalHash.get_AlgorithmName,System.Security.Cryptography,IncrementalHash,get_AlgorithmName(),0
M:System.Security.Cryptography.IncrementalHash.GetCurrentHash,System.Security.Cryptography,IncrementalHash,GetCurrentHash(),0
M:System.Security.Cryptography.IncrementalHash.AppendData(System.Byte[]),System.Security.Cryptography,IncrementalHash,AppendData(Byte[]),0
"M:System.Security.Cryptography.IncrementalHash.CreateHMAC(System.Security.Cryptography.HashAlgorithmName,System.Byte[])",System.Security.Cryptography,IncrementalHash,"CreateHMAC(HashAlgorithmName, Byte[])",0
M:System.Security.Cryptography.IncrementalHash.get_HashLengthInBytes,System.Security.Cryptography,IncrementalHash,get_HashLengthInBytes(),0
"M:System.Security.Cryptography.IncrementalHash.TryGetCurrentHash(System.Span{System.Byte},System.Int32@)",System.Security.Cryptography,IncrementalHash,"TryGetCurrentHash(Span<Byte>, Int32)",0
"M:System.Security.Cryptography.IncrementalHash.TryGetHashAndReset(System.Span{System.Byte},System.Int32@)",System.Security.Cryptography,IncrementalHash,"TryGetHashAndReset(Span<Byte>, Int32)",0
"M:System.Security.Cryptography.IncrementalHash.AppendData(System.Byte[],System.Int32,System.Int32)",System.Security.Cryptography,IncrementalHash,"AppendData(Byte[], Int32, Int32)",0
M:System.Security.Cryptography.IncrementalHash.GetHashAndReset,System.Security.Cryptography,IncrementalHash,GetHashAndReset(),0
M:System.Security.Cryptography.IncrementalHash.CreateHash(System.Security.Cryptography.HashAlgorithmName),System.Security.Cryptography,IncrementalHash,CreateHash(HashAlgorithmName),0
```

Reply:
Recently @akoeplinger has re-enabled `IncrementalHash` class for Browser in https://github.com/dotnet/runtime/pull/40486